pkg/yang/ast.go

```diff
@@ -143,13 +147,13 @@ func build(s *Statement, p reflect.Value, d *typeDictionary) (v reflect.Value, e
 
 	if fn := y.funcs["Name"]; fn != nil {
 		// Name uses s directly.
-		if err := fn(s, v, p); err != nil {
+		if err := fn(s, v, p, d); err != nil {
```

Comment:
I think the names here have got a bit too pithy -- can we call them something a bit more descriptive, I'd suggest:

```
s --> stmt
p --> parent
d --> types
```

`d` particularly doesn't make much sense when you're reading through.

Reply:
Done.